packages/flutter_localizations/lib/src/l10n/generated_date_localizations.dart

```diff
@@ -8396,6 +8396,184 @@ const Map<String, dynamic> dateSymbols = <String, dynamic>{
       r'''{1}, {0}'''
     ],
   },
+  'kn': <String, dynamic>{
+    'NAME': r'''kn''',
+    'ERAS': <dynamic>[r'''ಕ್ರಿ.ಪೂ''', r'''ಕ್ರಿ.ಶ'''],
```

Comment:
I assume that the Kannada characters used in `generated_date_localizations.dart` do not cause the emacs crash?

Reply:
Good catch! No, they're not safe. Will update.